src/components/app/artistscontainer/artistinfocontainer/artistinfo/artist-info.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import { observer, inject } from 'mobx-react'
+
+@inject('artistInfoStore')
+@observer
+class ArtistInfo extends Component {
+  render() {
+    const { artistInfoStore } = this.props
+
+    return (
+      <div className="field-name-container">
+        <span className="field-name">Info</span>
+        <div className="ai__artists-grid">
+          <div className="ai__name">
+            <span>{artistInfoStore.artistInfo.name}</span>
+          </div>
+          <div className="ai__popularity">
+            <span>Popularity: </span>
+            <span>{artistInfoStore.artistInfo.popularity}</span>
+          </div>
+          <div className="ai__followers">
+            <span>Followers: </span>
+            <span>{artistInfoStore.artistInfo.followers}</span>
```

Comment:
Destructure zamiast za każdym razem pisać artistInfoStore

Reply:
Nigdy nie wiem jak 'gleboko' robic ten destructure. Czy wywalic tylko artistInfoStore czy artistInfoStore.artistInfo zeby tylko bylo name, popularity, itd?

Zrobilem to drugie poki co bo w sumie dosc maly komponent, ale czasem jak jest wiekszy to lubie cos miec wiecej

Reply:
Yup jak wiadomo o jaki name i followers chodzi to można tak ;) jak są dwa różne name'y to wtedy warto zostawić